.serena/project.yml

```
@@ -1,3 +1,5 @@
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)